x/wasm/keeper/proposal_handler.go

```diff
@@ -68,7 +68,10 @@ func handleStoreCodeProposal(ctx sdk.Context, k types.ContractOpsKeeper, p types
 	if err != nil {
 		return err
 	}
-	return k.PinCode(ctx, codeID)
+	if p.PinCode {
```

Comment:
This would become `if !p.Unpinned` (or similar name)

Reply:
switched to early return instead
```
// if code should not be pinned return earlier
	if p.UnpinCode {
		return nil
	}
```
I think it has better readability, double negation logic confuses me a bit personally 😅   but I can change it if there is a preference.